go.mod

```diff
@@ -1,22 +1,204 @@
 module github.com/bringg/go_redis_ratelimit
 
-go 1.17
+go 1.21
 
 require (
-	github.com/alicebob/miniredis/v2 v2.16.1
+	github.com/alicebob/miniredis/v2 v2.31.1
 	github.com/coinpaprika/ratelimiter v0.2.1
 	github.com/go-redis/redis/v8 v8.11.4
```

Comment:
isn't this moved to `github.com/redis/go-redis`?

Reply:
I did another PR to upgrade go-redis, will send it to review after this PR is merged 🙏